forester/src/nullifier/nullify.rs

```diff
@@ -84,7 +82,8 @@ pub async fn nullify<T: Indexer>(indexer: T, config: &Config) -> Result<(), Fore
             .compressed_account_proofs
             .remove(&account.hash_string())
         {
-            let client = RpcClient::new(&config.server_url);
+            let client =
+                RpcClient::new_with_commitment(&config.server_url, CommitmentConfig::confirmed());
```

Comment:
what's the difference between RpcClient and RpcConnection?

Reply:
think we should make use of the `get_rpc_client_confirmed` helper that you added below